### PR TITLE
fix(coding):  update instructions to CSS Flexbox challenge in Portuguese

### PR DESCRIPTION
**Pull request by brenoxav**
Change the positions of some words to be in accordance with the syntax rules of the Portuguese language.

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/portuguese/01-responsive-web-design/css-flexbox/use-the-flex-shorthand-property.portuguese.md [hunk 1]
```diff
@@ -7,10 +7,13 @@ localeTitle: Use a propriedade flexível de taquigrafia
 ---
 
 ## Description
-<section id="description"> Existe um atalho disponível para definir várias propriedades flexíveis de uma só vez. As propriedades <code>flex-grow</code> , <code>flex-shrink</code> e <code>flex-basis</code> podem ser todas juntas usando a propriedade <code>flex</code> . Por exemplo, <code>flex: 1 0 10px;</code> irá definir o item para <code>flex-grow: 1;</code> <code>flex-shrink: 0;</code> e <code>flex-basis: 10px;</code> . As configurações de propriedade padrão são <code>flex: 0 1 auto;</code> . </section>
+<section id="description"> Existe um atalho disponível para definir várias propriedades *flex* de uma só vez. As propriedades <code>flex-grow</code> , <code>flex-shrink</code> e <code>flex-basis</code> podem ser definidas todas juntas usando a propriedade <code>flex</code> .
+Por exemplo, <code>flex: 1 0 10px;</code> irá definir o item para <code>flex-grow: 1;</code> <code>flex-shrink: 0;</code> e <code>flex-basis: 10px;</code> .
```
Comment (thefrontendwizard) [inline curriculum/challenges/portuguese/01-responsive-web-design/css-flexbox/use-the-flex-shorthand-property.portuguese.md]: Remove line break.

File: curriculum/challenges/portuguese/01-responsive-web-design/css-flexbox/use-the-flex-shorthand-property.portuguese.md [hunk 2]
```diff
@@ -7,10 +7,13 @@ localeTitle: Use a propriedade flexível de taquigrafia
...
+As configurações padrão da propriedade são <code>flex: 0 1 auto;</code> . </section>
```
Comment (thefrontendwizard) [inline curriculum/challenges/portuguese/01-responsive-web-design/css-flexbox/use-the-flex-shorthand-property.portuguese.md]: Remove line break.

File: curriculum/challenges/portuguese/01-responsive-web-design/css-flexbox/use-the-flex-shorthand-property.portuguese.md [hunk 3]
```diff
@@ -7,10 +7,13 @@ localeTitle: Use a propriedade flexível de taquigrafia
...
+<section id="instructions"> Adicione a propriedade CSS <code>flex</code> a ambos <code>#box-1</code> e <code>#box-2</code> . Dê a <code>#box-1</code> os valores para que sua <code>flex-grow</code> seja 2, sua <code>flex-shrink</code> seja 2 e sua <code>flex-basis</code> seja 150px. Dê a <code>#box-2</code> os valores para que sua <code>flex-grow</code> seja 1, sua <code>flex-shrink</code> seja 1 e sua <code>flex-basis</code> seja 150px.
+Esses valores farão com que <code>#box-1</code> cresça para preencher o espaço extra com o dobro da taxa de <code>#box-2</code> quando o container for maior que 300px e encolherá com o dobro da taxa de <code>#box-2</code> quando o container for menor que 300px. 300px é o tamanho combinado dos valores de <code>flex-basis</code> das duas caixas. </section>
```
Comment (thefrontendwizard) [inline curriculum/challenges/portuguese/01-responsive-web-design/css-flexbox/use-the-flex-shorthand-property.portuguese.md]: Remove line break.

**Review by frontendwizard — CHANGES_REQUESTED** [review]

**Comment by lynxlynxlynx** [comment]
@thefrontendwizard markdown doesn't care about those newlines, but they do improve readability of the source.

See for example the rendered output comparison — the newlines got eaten up: https://github.com/freeCodeCamp/freeCodeCamp/pull/19077/files?short_path=722e456#diff-722e456cb1a96bf49df457120e39683e

**Comment by RandellDawson** [comment]
@lynxlynxlynx Would you merge this PR?

**Comment by lynxlynxlynx** [comment]
Yes, it looks fine to me.

**Review by lynxlynxlynx — APPROVED** [review]

**Review by Briseno00 — COMMENTED** [review]
Nueva configuracion de roles

**Comment by lynxlynxlynx** [comment]
If you want to suggest changes to the current Portuguese text, please submit a new pull request with them — this here is not the right place.

**Comment by ilenia-magoni** [comment]
@lynxlynxlynx @Prisca1260 please do not translate through GitHub, translations are done through Crowdin

**Pull request merged**